src/store/modules/chat/index.js

```diff
@@ -558,6 +558,20 @@ const actions = {
       .then(({ messages, lastOffset }) => {
         dispatch('unshiftMessages', messages)
 
+        if (messages && offset === 0) {
+          dispatch(
+            'botCommands/initBotCommands',
+            {
+              partnerId: contactId,
+              commands: messages
+                .filter((item) => item.message.startsWith('/'))
```

Comment:
```suggestion
                .filter((item) => item.message?.startsWith('/'))
```

Message may be undefined